docs/tutorial/window-customization.md

```diff
@@ -1,42 +1,58 @@
+
+```fiddle docs/fiddles/features/window-customization/starter-code
 
-```js title='main.js'
-const { BrowserWindow } = require('electron')
-const win = new BrowserWindow({ frame: false })
 ```
 
-## Apply custom title bar styles _macOS_ _Windows_
+#### Removing the default titlebar
 
-Title bar styles allow you to hide most of a BrowserWindow's chrome while keeping the
-system's native window controls intact and can be configured with the `titleBarStyle`
-option in the `BrowserWindow` constructor.
+Let’s start by configuring a window with native window controls and a hidden titlebar. To remove the default titlebar, set the [`BaseWindowContructorOptions`][] `titleBarStyle` param in the `BrowserWindow` constructor to `'hidden'`.
 
-Applying the `hidden` title bar style results in a hidden title bar and a full-size
-content window.
+```fiddle docs/fiddles/features/window-customization/remove-titlebar
+
+```
+
+#### Adding native window controls _Windows_ _Linux_
+
+On macOS, setting `titleBarStyle: 'hidden'` removes the titlebar while keeping the window’s traffic light controls available in the upper left hand corner. However on Windows and Linux, you’ll need to add window controls back into your `BrowserWindow` by setting the [`BaseWindowContructorOptions`][] `titleBarOverlay` param in the `BrowserWindow` constructor.
+
+```fiddle docs/fiddles/features/window-customization/native-window-controls
+
+```
+
+Setting `titleBarOverlay: true` is the simplest way to expose window controls back into your `BrowserWindow`. If you’re interested in customizing the window controls further, check out the sections [Customizing traffic lights][] and [Customizing window controls][] that cover this in more detail.
+
+#### Create a custom titlebar
+
+Now, let’s implement a simple custom titlebar in the `webContents` of our `BrowserWindow`. There’s nothing fancy here, just HTML and CSS!
+
+```fiddle docs/fiddles/features/window-customization/custom-titlebar
+
+```
+
+#### Setting a custom drag region
+
+Currently our application window can’t be moved. Since we’ve removed the default titlebar, the application needs to tell Electron which regions are draggable. We’ll do this by adding the CSS style `app-region: drag` to the custom titlebar. Now we can drag the custom titlebar to reposition our app window!
+
+```fiddle docs/fiddles/features/window-customization/custom-drag-region
 
-```js title='main.js'
-const { BrowserWindow } = require('electron')
-const win = new BrowserWindow({ titleBarStyle: 'hidden' })
 ```
 
-### Control the traffic lights _macOS_
+For more information around how to manage drag regions defined by your electron application, see the [Custom draggable regions][] section below.
+
+Congratulations, you've just implemented a basic custom titlebar!
 
-On macOS, applying the `hidden` title bar style will still expose the standard window
-controls (“traffic lights”) in the top left.
+### Customizing traffic lights _macOS_
```

Comment:
Maybe adding another heading for advanced customization in the middle to mark that the basic tutorial section is over would be good